lib/tasks/importmap.rake

```diff
@@ -31,11 +31,13 @@ namespace :importmap do
     )
 
     if (imports = packager.import(*packages, **options))
-      imports.each do |package, url|
-        puts %(Verifying "#{package}" download from #{url})
-        packager.verify(package, url, verbose: ENV["VERBOSE"])
-        path = packager.vendored_package_path(package)
-        puts %(Verified  "#{package}" at #{path})
+      actual_imports = imports.is_a?(Hash) && imports.key?(:imports) ? imports[:imports] : imports
+      actual_imports.each do |package, url|
+        package_name = package.to_s
+        puts %(Verifying "#{package_name}" download from #{url})
+        packager.verify(package_name, url, verbose: ENV["VERBOSE"])
+        path = packager.vendored_package_path(package_name)
+        puts %(Verified  "#{package_name}" at #{path})
```

Comment:
@martinemde This logic broke on the [bump to Importmap-rails 2.2](https://github.com/rubygems/rubygems.org/pull/5838). I don't feel confident in this space, and I see you last edited this workflow, so I'd love it if you could take a look at this commit specifically. Here's the [diff from the breaking change](https://github.com/rails/importmap-rails/compare/v2.1.0...v2.2.0).